content/chat/getting-started/javascript.textile

```diff
@@ -0,0 +1,195 @@
+---
+title: "Getting started: Chat and JavaScript"
+meta_description: "A getting started guide for Ably Chat that steps through some of the key features using JavaScript."
+meta_keywords: "Ably, realtime, quickstart, getting started, basics, chat"
+---
+
+This guide will get you started with Ably Chat.
+
+It will take you through the following steps:
+
+* Complete the prerequisites before you begin.
```

Comment:
Can we be specific about what this is? "prerequisites" sounds like "a big list of things i have to do to get started"